compiler/noirc_frontend/src/parser/parser.rs

```diff
@@ -370,6 +370,10 @@ pub fn trait_bound() -> impl NoirParser<TraitBound> {
     traits::trait_bound()
 }
 
+pub fn path_no_turbofish() -> impl NoirParser<Path> {
+    path::path_no_turbofish()
+}
+
```

Comment:
```suggestion
```
Can you make the `path` and `traits` module `pub(super)` so that we don't have to copy these functions to re-export them from here? Then we can remove this `path_no_turbofish` copy as well as the `trait_bound` copy.

Reply:
Done! Let me know if I got this wrong.